### ISSUE TITLE
On description.ext import got java.lang.IllegalArgumentException: workspaceDirectory doesn't exist

### ISSUE THREAD

**Issue by kju** [kayler-renslow/arma-dialog-creator#32]
### Version and Build
1.0.5 - b33

### Summary
Installed latest version, created workspace when asked and then tried to import.
After selecting the .exe, said error was thrown.

### Detail
Local path selected is:
> F:\workspaceArma\Blitzkrieg_Old\dev\TM.Limnos

```
java.lang.IllegalArgumentException: workspaceDirectory doesn't exist
	at com.kaylerrenslow.armaDialogCreator.data.Workspace.<init>(Workspace.java:29)
	at com.kaylerrenslow.armaDialogCreator.data.HeaderToProject.convertAndSaveToWorkspace(HeaderToProject.java:65)
	at com.kaylerrenslow.armaDialogCreator.gui.main.popup.projectInit.ImportDialogsDialog$ConvertTask.call(ImportDialogsDialog.java:137)
	at com.kaylerrenslow.armaDialogCreator.gui.main.popup.projectInit.ImportDialogsDialog$ConvertTask.call(ImportDialogsDialog.java:125)
	at javafx.concurrent.Task$TaskCallable.call(Task.java:1423)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

THREAD:Arma Dialog Creator JavaFX Thread
```

**Comment by kju** [comment]
It appears no workspace was created yet.
When i made a test project first, closed that project and did the import again, it worked

**Comment by kayler-renslow** [comment]
fixed in https://github.com/kayler-renslow/arma-dialog-creator/commit/f1c041f2ccfb1825ac9fafabd873b0dbad6bda60

**Comment by kayler-renslow** [comment]
fixed in release 1.0.6
